### ISSUE TITLE
Add running-time of CNN models in the document

### ISSUE THREAD

**Issue by BbChip0103** [keras-team/keras#15004]
Hi, I have a interesting suggestion.

In the [Keras application document](https://keras.io/api/applications/), It describes model's size, accuracy, etc.
When we use the model for our real world project, 'running-time' is most important feature like 'accuracy'.
Of course, running-time generally tend to follow number of parameters. But these are not perfectly proportional.
So checking the running-time of models will be useful when we search model for our project.
As the running-time depends on the spec of machine, we can use fixed and general environment for the checking (like google-colab)
 
How about this idea? 
If peoples love this idea, I'm willing to do this.

**Comment by tomerk** [comment]
Hi @BbChip0103, if you have a reference GPU + CPU to run these models on this could be useful to have. If you use google colab you should make sure it's using dedicated reference hardware & resources across all runs (w/o e.g. throttling or sharing resources with other colab users that may affect your own performance)

If you do it'd be good to measure inference step time (not training time), and run the models on a few images first to warm them up.

**Comment by BbChip0103** [comment]
Thank you for the reply @tomerk .
I checked the inference time of models like your advice using google colab environment.
Checking was repeated 30 times and averaged for each model.
Here is the result and the [colab link](https://colab.research.google.com/drive/19PGl68orniUzHsFEwvLT6xumoEUjVNeN?usp=sharing)
![time checking result](https://i.ibb.co/7YYSgjT/plot.png)

If it has no problem.  then i want to update the keras document.
What do you think about this?

**Comment by tomerk** [comment]
Hi @BbChip0103 , feel free to open a PR to the Keras documentation and add details on the GPU & system specs that were used in your colab for inference.

**Comment by BbChip0103** [comment]
Merged [keras-team/keras.io PR#603](https://github.com/keras-team/keras-io/pull/603)

Thank you!
